packages/wallets/src/features/cfd/screens/CompareAccounts/CompareAccountsTitleIcon.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useRef } from 'react';
+import { useHover } from 'usehooks-ts';
+import { Tooltip, WalletText } from '../../../../components';
+import InfoIcon from '../../../../public/images/ic-info-outline.svg';
+import TradingPlatformIcons from '../../../../public/images/tradingPlatforms';
+import { CFD_PLATFORMS, MARKET_TYPE, MARKET_TYPE_SHORTCODE } from './constants';
+import './CompareAccountsTitleIcon.scss';
+
+type TPlatforms = typeof CFD_PLATFORMS[keyof typeof CFD_PLATFORMS];
+// type TShortCode = 'bvi' | 'labuan' | 'maltainvest' | 'svg' | 'vanuatu';
```

Comment:
@arshad-rao-deriv delete this too please if you don't need this

Reply:
Updated.